kubernetes/argocd/stacks/common/cert-manager/self-ca.yml

```diff
@@ -35,7 +38,8 @@ kind: Certificate
 metadata:
   name: test-cert
 spec:
-  commonName: test.acmuic.org
+  dnsNames:
```

Comment:
Here, we remove the `commonName` attribute, as this is no longer used by modern web browsers to determine the name of the domain the TLS certificate is securing.

For more information, see https://cert-manager.io/docs/usage/certificate/:
> The use of the common name field has been deprecated since 2000 and is
> discouraged from being used.